AppCenter/AppCenter/MSACAppCenter.h

```diff
@@ -109,6 +109,11 @@ NS_SWIFT_NAME(AppCenter)
  */
 @property(class, nonatomic, strong) NSString *logUrl;
 
+/**
+ * Data residency region.
```

Comment:
Here you need to expand the annotation, as stated in the description of the work item.

Reply:
Corrected